src/game.cpp

```diff
@@ -13859,6 +13857,10 @@ void game::add_artifact_messages(std::vector<art_effect_passive> effects)
         case AEP_SICK:
             add_msg(m_bad, _("You feel unwell."));
             break;
+
+        case AEP_SMOKE:
+            add_msg(m_warning, _("The %s begins to smoke..."));
```

Comment:
Yes, sorry. Adding the %s was a last-minute bit - originally it was just "Smoke begins to rise...", figures it would break everything.